.github/workflows/cuda.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         which nvcc || echo "nvcc not in PATH!"
 
         git clone https://github.com/AMReX-Codes/amrex.git ../amrex
-        cd amrex && git checkout --detach 6d30e83c944e4e6167178f2d145df2a3e67d2b24 && cd -
```

Comment:
Please dbl check the script in `/home/axel/src/warpx/Tools/Release/updateAMReX.py` updates this location correctly :)